### PR TITLE
Newcomers_Guide.rst: Specified the cobot assign <issue_link> command

### PR DESCRIPTION
**Pull request by CosminIcatoiu**
Added a short paragraph to Step 3 of the Newcomers_Guide in which it
is clearly specified what a newcomer should do in order to get an
issue. Tried to do it in a friendly and entertaining way. It was a
needed change because as it was in my case there were many newcomers
who forgot that they shouldn`t work unassigned on an issue and also
there were newcomers who didn`t know how to assign themselves to an
issue and there appeared a lot of not really necessary questions on
chat.

Fixes https://github.com/coala/coala/issues/3495

### Checklist

- [x] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
- [x] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
- [x] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
- [x] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
- [x] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
- [x] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
as gitmate pointed out, you should run "coala" over your project and "amend" your last commit using "git commit --amend" and push again, this time with "--force".

**Comment by Adrianzatreanu** [comment]
1. the first line of your commit message is too long, must be 50 maximum
2. it should be in imperative tense, aka "Specify"

**Comment by harshit890** [comment]
@CosminIcatoiu before u go for ur next commit do read https://api.coala.io/en/latest/Developers/Writing_Good_Commits.html

**Comment by CosminIcatoiu** [comment]
Read it now. Thanks. Hope it will be ok this time :-D

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -141,6 +141,14 @@ Now you can start working on it.
 
 .. note::
 
+    As stated before, you should never work on an issue without any
+    assignment. Fortunately, cobot is here to help you! So, if you are
+    interested in picking up an issue just write in the gitter chat the
+    following command:
+
+        ``cobot assign <issue_link>``
```
Comment (Nosferatul) [inline docs/Developers/Newcomers_Guide.rst]: I believe you should add a blank line after this

Comment (CosminIcatoiu) [inline docs/Developers/Newcomers_Guide.rst]: Thanks for the advice. I try to figure out if there is something else wrong :) before i commit again

**Comment by sims1253** [comment]
Typo in commit shortlog (first line) commamd

**Comment by sims1253** [comment]
Also this is not a bugfix so change the Fixes in your commit message to a Closes

File: docs/Developers/Newcomers_Guide.rst [hunk 2]
```diff
@@ -141,6 +141,15 @@ Now you can start working on it.
 
 .. note::
 
+    As stated before, you should never work on an issue without any
+    assignment. Fortunately, cobot is here to help you! So, if you are
+    interested in picking up an issue just write in the gitter chat the
+    following command:
+
```
Comment (aptrishu) [inline docs/Developers/Newcomers_Guide.rst]: Let it look like a command. It'll look better this way -  
::

    cobot assign <issue_link>

**Comment by aptrishu** [comment]
Also Please shorten your commit body a bit. It doesn't need that much explanation, not a major change.. :P

**Comment by sims1253** [comment]
well a long commit body doesn't really hurt anyone :D

**Comment by CosminIcatoiu** [comment]
I think i changed all you said. Hope it will be okay this time :)

File: docs/Developers/Newcomers_Guide.rst (hunk 2 above)
Comment (CosminIcatoiu) [inline docs/Developers/Newcomers_Guide.rst]: I changed it . Thanks for advice :)

Comment (aptrishu) [inline docs/Developers/Newcomers_Guide.rst]: an extra newline is needed between :: and the command.

Comment (CosminIcatoiu) [inline docs/Developers/Newcomers_Guide.rst]: Ok. I`ll do it now.

File: docs/Developers/Newcomers_Guide.rst [hunk 3]
```diff
@@ -63,7 +63,7 @@ As a preparation of joining the community you should find out what this project
 is about - if you didn't do this already. We highly recommend you
 `install coala <https://coala.io/install>`_ and use it on at least one of your
 projects. Also, we recommend that you read
-`Development Setup Notes <http://coala.io/devsetup>`
+`development setup notes <http://coala.io/devsetup>`_
```
Comment (sils) [inline docs/Developers/Newcomers_Guide.rst]: I think this change happened in another PR and it's unrelated, that may lead to conflicts

File: docs/Developers/Newcomers_Guide.rst (hunk 2 above)
Comment (sils) [inline docs/Developers/Newcomers_Guide.rst]: I think you can also write:

```rst
following command::

    code here
```

with the double colon right there

Comment (CosminIcatoiu) [inline docs/Developers/Newcomers_Guide.rst]: I will change it now :) Thanks

**Comment by rhemon** [comment]
ack 93cdcde

LGTM 👍

**Comment by CosminIcatoiu** [comment]
Sorry to ask, but where does ack come from?

**Comment by rhemon** [comment]
ack 93cdcde

**Comment by rhemon** [comment]
guessing from gitmate. http://api.coala.io/en/latest/Developers/Review.html

**Comment by CosminIcatoiu** [comment]
Thanks :)

**Comment by sils** [comment]
ok this needs one last rebase, then we can merge it, ping me after the rebase please! :)

**Comment by CosminIcatoiu** [comment]
What`s a rebase and how should i do it?

**Comment by sils** [comment]
(I cancelled your build because they queued up!)

**Comment by sils** [comment]
`Closes #3500` your commit closes the PR? Why not the issue? a9c95dd needs work

**Comment by sils** [comment]
but the rebase was correct! :)

**Comment by sils** [comment]
you still reference the PR, not the issue

**Comment by CosminIcatoiu** [comment]
but i linked the whole issue . hmm :-?

**Comment by sils** [comment]
you linked this pull request!

Sincerely,

Lasse Schuirmann

lasse@schuirmann.net
http://coala.io/ - http://viperdev.io/ - http://gitmate.com/

**Comment by CosminIcatoiu** [comment]
I think i solved it now :) Sorry for the inconvenience

File: docs/Developers/Newcomers_Guide.rst [hunk 4]
```diff
@@ -63,7 +63,7 @@ As a preparation of joining the community you should find out what this project
 is about - if you didn't do this already. We highly recommend you
 `install coala <https://coala.io/install>`_ and use it on at least one of your
 projects. Also, we recommend that you read
-`development setup notes <http://coala.io/devsetup>`_
+`Development Setup Notes <http://coala.io/devsetup>`
```
Comment (sils) [inline docs/Developers/Newcomers_Guide.rst]: now you included this again? :/

Comment (sils) [inline docs/Developers/Newcomers_Guide.rst]: oh you actually break this here

**Comment by sils** [comment]
and one rebase, then we should be finally done. Generally: always rebase when you fix something up. (I usually use the `rebase -i` to fix up all my stuff and rebase as a side effect without noticing)

File: docs/Developers/Newcomers_Guide.rst (hunk 4 above)
Comment (CosminIcatoiu) [inline docs/Developers/Newcomers_Guide.rst]: Yea.  Forgot to change it second time .

**Comment by CosminIcatoiu** [comment]
Think i did it :)

**Comment by sils** [comment]
ack https://github.com/coala/coala/pull/3500/commits/28c292d4b9506ca3281367c3e6f58a84e75d5b46

**Comment by Makman2** [comment]
ack 5fef490

**Comment by Makman2** [comment]
Cherry-picked into https://github.com/coala/coala/pull/3520

**Pull request closed without merging**